doc/nrf/releases_and_maturity/releases/release-notes-3.0.0-preview1.rst

```diff
@@ -133,7 +133,9 @@ Bluetooth® LE
 Bluetooth Mesh
 --------------
 
-|no_changes_yet_note|
+* Added:
```

Comment:
The changes are still in the wrong file. Please, move them to the changelog in main.

Reply:
Hi @peknis, sorry, I do not understand what do you mean under "to the changelog in main". I'm not so familiar with documentation terms and structure. Could you provide exact place by link?

Reply:
Here https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst